lessons/operator/03/README.md

```diff
@@ -375,208 +217,446 @@ $ sensuctl configure -n \
+   Platform Version:       6.2.7
+   Auto-Deregistration:    true
+   Deregistration Handler: 
    ```
 
-1. **Export resources for a single namespace.**
+1. **Output Entity Information in JSON Format.**
+
+   Entity information is commonly used in automation scenarios, such as scripts or CI/CD workflows.
+   In those situations it you may wish to output this information in a structured format like JSON. 
 
    ```shell
-   sensuctl dump all --namespace default
-   ```
+   sensuctl entity info learn.sensu.io --format json
+   ```
+
+   **Example Output:**
+   ```json
+   {
+     "entity_class": "proxy",
+     "system": {
+       "hostname": "learn.sensu.io",
+       "os": "Workshop",
+       "platform": "Sensu Go",
+       "platform_family": "Training",
+       "platform_version": "6.2.7",
+       "network": {
+         "interfaces": [
+           {
+             "name": "lo",
+             "mac": "00:00:00:00:00:00",
+             "addresses": [
+               "127.0.0.1/8",
+               "::1/128"
+             ]
+           },
+           {
+             "name": "eth0",
+             "mac": "00:00:00:00:00:00",
+             "addresses": [
+               "10.0.0.1/8"
+             ]
+           }
+         ]
+       },
+       "arch": "arm",
+       "libc_type": "",
+       "vm_system": "",
+       "vm_role": "",
+       "cloud_provider": "",
+       "processes": null
+     },
+     "subscriptions": [
+       "entity:learn.sensu.io",
+       "workshop-test"
+     ],
+     "last_seen": 0,
+     "deregister": true,
+     "deregistration": {},
+     "user": "workshop",
+     "metadata": {
+       "name": "learn.sensu.io",
+       "namespace": "default",
+       "labels": {
+         "app": "workshop"
+       },
+       "created_by": "sensu"
+     },
+     "sensu_agent_version": ""
+   }
+   ```
+
+## Monitoring as Code
+
+The `sensuctl` monitoring-as-code workflow manages resource configurations using plain-text formats like YAML and JSON.
+
+Managing resources follows these basic steps:
+1. Resource configurations are defined in a YAML or JSON file.
+2. The `sensuctl create` command reads the resource configuration from the file, then pushes that configuration to the backend API where the resource is created.
+3. The file can then be saved to a file within a source code repository. 
+4. Later, the file can be modified and `sensuctl create` is used to update the resource.
+
+You can also use the `sensuctl dump` command to export a resource to a file, or add the `--format` option to a `sensuctl entity info` command to output the resource in a structured format. 
+
+These features together enable Sensu's _monitoring-as-code_ workflow.
+
+### EXERCISE 6: Exporting and Updating Resource Configurations
+#### Scenario
+
+You have an entity in a running Sensu environment but don't have a YAML file that describes it. You'd like to save the resource to a file so you can modify the configuration, and store that configuration to a code repository.
```

Comment:
Entities are probably not the right resource to use for this scenario (because in some cases, the Sensu API is not the source of truth for an entity – [the Sensu Agent is](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-entities/entities/#manage-agent-entities-via-the-agent)). 

An alternative example could be that you are onboarding new team members who have created/modified checks using the Sensu web app (sidenote: this is just another API client) that you now wish to bring under management as code... then go through more-or-less the same workflow you have in the exercise. 

Sorry I didn't catch this sooner!

Reply:
Yeah, I thought about this, but at this stage in the workshop, we really don't have much else to work with that makes sense here. I think this is relatively benign and it gently leads into the topic of subscriptions which we will be getting more into later. I think for now, let's leave this as-is, and maybe update the workshop environment to seed it with a resource that might be more interesting to use for this.